src/lib/components/cards/a.stories.mdx

```diff
@@ -19,15 +11,15 @@ Cards are interactive elements that can be used to display an action that a user
     width="100%"
     height="450"
     src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FMKs4cbojdVOBKUxv7okb93%2FDirty-Swan-Pattern-Library%3Fnode-id%3D358%253A19"
-    allowfullscreen
-  ></iframe>
+    allowFullScreen
+  />
 </Preview>
 
 # Models
 
-## Title Size (small, medium, big)
+## Title Size (xsmall, small, medium, big)
 
-The card title can be set with the following properties: `small`, `medium`, `big`.
+The card title can be set with the following properties: `xsmall`, `small`, `medium`, `big`.
 
 ## Title Size (medium, big)
```

Comment:
I think this line and line 34 can be removed as the descriptions above are added

Reply:
Got it!